jetbrains-core/src/software/aws/toolkits/jetbrains/core/stack/ToolWindow.kt

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.core.stack
+
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.wm.ToolWindowAnchor
+import com.intellij.openapi.wm.ToolWindowManager
+import com.intellij.ui.content.impl.ContentImpl
+import javax.swing.JComponent
+
+internal class ToolWindow(
+    private val component: JComponent,
+    private val project: Project,
+    private val stackName: String,
+    private val toolWindowId: String,
+    private vararg val disposables: Disposable
+) {
+
+    fun show() {
+        val contentManager = window.contentManager
```

Comment:
Can we make this additive? So we could monitor 2 stacks at once as different Contents in the tool window? Then close it when all Contents are disposed? Like Find window

Reply:
I think yes, will try to do that.

Reply:
Done. We now have multuple tabs.